Slick Next / Previous buttons

RojoRevolution opened the pull request:
Updated buttons for the previous / next buttons. There should be no changes to anything outside of the slick-AddOn.CSS file.

- two new .svg files are getting added to the images folder
- Adjusted the the position right / left of .slick-next and .slick previous
- added width and height to both .slick-next and .slick-previous
- Added and overwrote the Content property in classes from lines 35-53

RojoRevolution committed:
added properties to :before classes and overwrote positions on slick-next and slick-previous

RojoRevolution committed:
overwrote rules in slick-addOn CSS, replaced content with actual SVG files

RojoRevolution committed:
changed color of SVG files, adjusted height, width, and position of slick-prev and slick-next classes

reagang1997 reviewed (changes requested):
This is creating a pretty complex merge in .DS_Store

Is there a way around this?

RojoRevolution committed:
Delete .DS_Store

This file is created and used by Mac OSX - It shouldn't be needed for anything on gitHub

Deleting to see if it fixes the merge
